README.md

```diff
@@ -28,7 +28,7 @@
 - [📝 License](#license)
 
 
-**Blog App** In this project, I create all controllers and views for my Blog app. As I am not processing any data yet, I add simple placeholders for your views with plain HTML.
```

Comment:
![image](https://github.com/eamador1/blog-app/assets/93039359/71e08851-05c3-44bb-83c2-8802d953c12c)

- [ ] Great work but here should be the title of the project to make your readme descriptive and professional 👍